app/src/main/java/ch/epfl/sweng/swenggolf/offer/ShowOfferActivity.java

```diff
@@ -408,9 +409,21 @@ public void postAnswer(View view) {
                         new Notification(NotificationType.ANSWER_POSTED,
                                 Config.getUser(), offer));
             }
+            sendNotificationToPreviousAnswerers(answers.getAnswerList());
             listAnswerAdapter.notifyDataSetChanged();
         }
+    }
 
+    private void sendNotificationToPreviousAnswerers(List<Answer> answerList) {
+        List<String> participants = new ArrayList<>();
```

Comment:
In this case, you're only adding and testing if the collection contains data. A set would be more efficient for this kind of usage.

Reply:
This is absolutely correct thanks.